tests/django/test_views.py

```diff
@@ -329,3 +329,31 @@ def abc(self, info: Info) -> str:
 
     assert response.status_code == 418
     assert data == {"data": {"abc": "ABC"}}
+
+
+def test_print_context(mocker):
+    # Swallow the print statements to keep the logs clean.
+    mock = mocker.patch("builtins.print")  # noqa: F841
+    factory = RequestFactory()
+
+    @strawberry.type
+    class Query:
+        @strawberry.field
+        def abc(self, info) -> str:
+            print(info.context.request)
+            print(info.context)
```

Comment:
Can you add some assertions on the mock print to check that it's behaving as expected?

Reply:
Sure thing, I'll update it accordingly

Reply:
Updated, thanks for the feedback!